### PR TITLE
fix(frontend): missing team details navigation

### PR DESCRIPTION
**Pull request by moncapitaine**

### CODE REVIEW

**Comment by moncapitaine** [comment]
image

File: app/backend/graphql/interfaces/modifyInterface.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const ModifyInterface = interfaceType({
         }
 
         if (!context.teamSlug) {
-          throw new GraphQLError('team slug not found')
+          throw new GraphQLError('team slug not found', {})
```
Comment (YohannesTesfay) [inline app/backend/graphql/interfaces/modifyInterface.ts:17]: What does this `{}` do? It fixes the depreciated GraphQLError?

File: app/frontend/components/teamTile/teamTile.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import Link from 'next/link'
+import { TeamWithProjectsFragment } from '../../generated/graphql'
+
+export interface TeamTileProps {
+  team: TeamWithProjectsFragment
+}
+export const TeamTile = ({ team }: TeamTileProps): JSX.Element => {
```
Comment (YohannesTesfay) [inline app/frontend/components/teamTile/teamTile.tsx:7]: If I am not mistaken, you had already implemented this new `teamTile `in the older `teamChoice `component.

File: app/backend/graphql/interfaces/modifyInterface.ts (hunk 1 above)
Comment (moncapitaine) [reply app/backend/graphql/interfaces/modifyInterface.ts:17]: exactly

**Review by YohannesTesfay — APPROVED** [review]

File: app/frontend/components/teamTile/teamTile.tsx (hunk 2 above)
Comment (moncapitaine) [reply app/frontend/components/teamTile/teamTile.tsx:7]: I see no other TeamTile, gimme a hint?

**Pull request merged**